src/bin/help.ts

```diff
@@ -0,0 +1,230 @@
+import chalk from "chalk";
+
+import { logLine } from "../shared/cli/lines.js";
+
+export function logHelpText(version: string): void {
+	logLine();
+
+	logLine(chalk.cyan(`✨ template-typescript-node-package v${version} ✨`));
+
+	logLine(
+		chalk.cyan(
+			`
+      A quickstart-friendly TypeScript template with comprehensive formatting, 
+      linting, releases, testing, and other great tooling built-in.
+      `,
+		),
+	);
```

Comment:
Thinking on the sentences in this output:

```plaintext
┌  Welcome to create-typescript-app ! 🎉
│
│  ⚠️ This template is early stage, opinionated, and not endorsed by the TypeScript team. ⚠️
│  ⚠️ If any tooling it sets displeases you, you can always remove that portion manually. ⚠️
│
│  ✨ template-typescript-node-package v1.29.30 ✨
│  
      A quickstart-friendly TypeScript template with comprehensive formatting, 
      linting, releases, testing, and other great tooling built-in.
```

Seems a little duplicative to mention the title twice. Though I do like the new emojis 😄.

What do you think about:

* Having the CLI always log the version _(why not?)_
* Removing the second mention of the package name

Reply:
I think always logging the CLI version sounds good!